pypeit/metadata.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, spectrograph, par, files=None, data=None, usrdata=None, stric
                                  else self._build(files, strict=strict, usrdata=usrdata))
 
         # Sort on filename
-        self.table.sort('filename')
+        #self.table.sort('filename')  # JXP -- This was causing issues with merging, and I think we sort on MJD later
```

Comment:
This is fine with me as long as the output pypeit file is sorted by MJD.

Reply:
The MJD sort has not been removed.